test/e2e/adminUI/tests/group005Fields/url/uxTestUrlField.js

```diff
@@ -0,0 +1,83 @@
+module.exports = {
+	before: function (browser) {
+		browser.app = browser.page.app();
+		browser.signinPage = browser.page.signin();
+		browser.listPage = browser.page.list();
+		browser.itemPage = browser.page.item();
+		browser.initialFormPage = browser.page.initialForm();
+
+		browser.app.navigate();
+		browser.app.waitForElementVisible('@signinScreen');
+
+		browser.signinPage.signin();
+		browser.app.waitForElementVisible('@homeScreen');
+	},
+	after: function (browser) {
+		browser.app.signout();
+		browser.end();
+	},
+	'Url field can be filled via the initial modal': function (browser) {
+		browser.app
+			.click('@fieldsMenu')
+			.waitForElementVisible('@listScreen')
+			.click('@urlsFieldsSubmenu')
+			.waitForElementVisible('@listScreen');
+
+		browser.listPage
+			.click('@createFirstItemButton');
+
+		browser.app
+			.waitForElementVisible('@initialFormScreen');
+
+		browser.initialFormPage.section.form.section.urlList.section.name
+			.fillInput({value: 'Name Field Test 1'});
+
+		browser.initialFormPage.section.form.section.urlList.section.name
+			.verifyInput({value: 'Name Field Test 1'});
+
+		browser.initialFormPage.section.form.section.urlList.section.fieldA
```

Comment:
so it lets you save urls of this format:  'Url Field Test 1'?  may as well just be a text field :-)  but hmmm...if it's not validating url formats perhaps you should create a keystone issue for that.  Seems to me like it should...not sure.